src/app/containers/PageHandlers/__snapshots__/withData.test.jsx.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`withData HOC with missing articleData should return the errorMain component 1`] = `
+<ErrorMain
+  status={200}
+/>
+`;
+
+exports[`withData HOC with no data should return the errorMain component and 500 status 1`] = `
+<ErrorMain
+  status={500}
+/>
+`;
+
+exports[`withData HOC with non 200 status should return the errorMain component 1`] = `
+<ErrorMain
+  status={157}
+/>
+`;
+
+exports[`withData HOC with valid props should return the passed in component 1`] = `
```

Comment:
Is this the expected output for this test?

Reply:
ahhh, against incorrect props in the test data. 🙈